SofaKernel/framework/sofa/helper/vector.cpp

```diff
@@ -51,6 +52,62 @@ void SOFA_HELPER_API vector_access_failure(const void* vec, unsigned size, unsig
     assert(i < size);
 }
 
-} // namespace helper
+/// Convert the string 's' into an unsigned int. The error are reported in msg & numErrors
+/// is incremented.
+int SOFA_HELPER_API getInteger(const std::string& s, std::stringstream& msg, unsigned int& numErrors)
+{
+    const char* attrstr=s.c_str();
```

Comment:
I tested this version and it is very much slower. 

strtol: benchmark/vector_benchmark_int.benchmark/7 (8867 ms)
stringstream: benchmark/vector_benchmark_int.benchmark/7 (49733 ms) 

This surprise me a lot (I wonder if I made something wrong) but I have no time now to investigate so I will keep the current version, just adopting the early exit approach.